src/centralserver/registration-service/build.gradle

```diff
@@ -1,30 +1,84 @@
 plugins {
```

Comment:
I think we could have a `registration-service/README.md` that would shortly describe the role of the module. I believe someone suggested this kind of convention recently, and I think it would be useful when navigating the codebase.